### PR TITLE
As/starrating

### PR DESCRIPTION
**Pull request by Adrian-St**
Added Star Rating

### CODE REVIEW

File: app/views/exercises/show.html.slim [hunk 1]
```diff
@@ -142,32 +157,35 @@ table.vertical-table
               = t('exercises.tests.framework')
             td
               = test.testing_framework.name
+
+.actions.btn-group[role="group"]
+  = link_to 'Add to cart', add_to_cart_path(@exercise), class: 'btn btn-important'
```
Comment (tstaubitz) [inline app/views/exercises/show.html.slim]: please replace hard-coded string with locale

File: app/views/exercises/show.html.slim [hunk 2]
```diff
@@ -142,32 +157,35 @@ table.vertical-table
...
+  .dropup.btn-group
+    = button_tag class: 'btn btn-important dropdown-toggle',  data: {toggle: 'dropdown'} do
+      = 'Add to Collection '
```
Comment (tstaubitz) [inline app/views/exercises/show.html.slim]: same here

File: app/views/exercises/show.html.slim [hunk 3]
```diff
@@ -142,32 +157,35 @@ table.vertical-table
...
+      span.caret
+    ul.scrollable.dropdown-menu[role="menu"]
+      - if !current_user.collections.empty?
+        - current_user.collections.each do |collection|
+          li
+            = link_to collection.title, add_to_collection_path(collection: collection), method: 'post', class: 'dropdown-item'
+      - else
+        li
+          = link_to 'Add Collection', new_collection_path(current_user)
```
Comment (tstaubitz) [inline app/views/exercises/show.html.slim]: and here

File: app/views/exercises/show.html.slim [hunk 4]
```diff
@@ -142,32 +157,35 @@ table.vertical-table
...
-      th
-        = t('exercises.show.add_collection')
-      td
-        - if !current_user.collections.empty?
-          .btn-group[role="group"]
-            = collection_select(:collection, :id, Collection.where(user: current_user), :id, :title, {}, {class: 'btn btn-default dropdown-toggle'})
-            = button_tag 'Add to Collection', class: 'btn btn-main btn-sm'
+    .dropup.btn-group
+      = button_tag class: 'btn btn-important dropdown-toggle', data: {toggle: 'dropdown'} do
+        = 'Export '
```
Comment (tstaubitz) [inline app/views/exercises/show.html.slim]: and here

**Comment by rteusner** [comment]
apart of the missing i18n, lgtm

**Pull request merged**